nomad-aws/nomad-autoscaler.tf

```diff
@@ -38,4 +38,12 @@ resource "aws_iam_role" "nomad_role" {
     })
   }
   tags = local.tags
-}
\ No newline at end of file
+}
+
+resource "aws_iam_role_policy" "nomad_nodegroup_iam_role_policy" {
+  count = var.create_nomad_nodegroup_iam_role_policy ? 1 : 0
```

Comment:
We need a flag for the count here since we cannot depend on a computed variable. Otherwise, we'll need to do a `terraform apply` in two steps, which isn't great.

Reply:
cc @atulsingh0